example-projects/default-config/doc/spec.md

```diff
@@ -0,0 +1,6 @@
+# Tracing Example
+`dsn~example~1`
+
+Example requirement
+
+Needs: utest, impl
```

Comment:
Is it intentional that there is no difference to the file in the "custom-dir"?

Used in any test cases where a distinction is necessary?